File: ios/App/App/StatusBarPlugin.m
//
//  StatusBarPlugin.m
//  App
//
//  Created by Kyoung-Min Roh on 10/2/23.
//

#import <Foundation/Foundation.h>
#import <Capacitor/Capacitor.h>

CAP_PLUGIN(StatusBarPlugin, "MyStatusBar",
           CAP_PLUGIN_METHOD(getHeight, CAPPluginReturnPromise);
);

File: ios/App/App/StatusBarPlugin.swift
//
//  StatusBarPlugin.swift
//  App
//
//  Created by Kyoung-Min Roh on 10/2/23.
//

import Foundation
import Capacitor

@objc(StatusBarPlugin)
public class StatusBarPlugin: CAPPlugin {
    
    @objc func getHeight(_ call: CAPPluginCall){
        call.resolve([
            "height": 23
        ])
    }
}
